docs/ABI/Mangling.rst

```diff
@@ -146,7 +146,8 @@ Globals
   global ::= type protocol-conformance 'WL'   // lazy protocol witness table cache variable
 
   global ::= protocol-conformance identifier 'Wt' // associated type metadata accessor (HISTORICAL)
-  global ::= protocol-conformance assoc-type-list nominal-type 'WT' // associated type witness table accessor
+  global ::= protocol-conformance assoc-type-list protocol 'WT' // associated type witness table accessor
+  global ::= protocol-conformance type protocol 'WT' // inherited protocol witness table accessor
```

Comment:
Is there no ambiguity here?

Also, why do we need to define an accessor for this?  Has the resilient code generation for accessing a base protocol changed from just a load?

Reply:
The accessor is there so we can call it to fill in the witness table entry, and is referenced via the mangled name (the same as we do for associated conformances). This moves the code from the instantiation function into the accessor.

The resilient code generation for accessing a base protocol has not changed, because the runtime forces the witness table entries for base protocols to be realized during instantiation.

Reply:
It's an internal-linkage function that returns the right witness table.